ScrollViewAssignment/ScrollViewAssignment/View Controllers/TableViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  TableViewController.swift
+//  ScrollViewAssignment
+//
+//  Created by Стожок Артём on 10.08.2021.
+//
+
+import UIKit
+
+class TableViewController: UITableViewController {
+    
+    @IBOutlet weak var addButton: UIBarButtonItem!
+    
+    lazy var choosingItems: [ShopItem] = {
```

Comment:
this things should be private unless it is not needed somewhere else, check all functions and variables and set all to private